test/broadway_test.exs

```diff
@@ -1184,6 +1184,9 @@ defmodule BroadwayTest do
 
       assert processor_1 != processor_2
       assert batch_processor_1 != batch_processor_2
+
+      # NOTE: we need to stop this one because is not stopping alone and causing intermittence.
+      GenServer.stop(broadway)
```

Comment:
Perhaps a better fix is rather to start the topology using `start_supervised`. Then ExUnit will guarantee to stop it.

Reply:
I had to create a new module because I was not getting the `child_spec` right. In the end it seems easier to read this way. WDYT?

Reply:
Sounds good to me. If you don’t want to create a module, you can also define a map child_spec, maybe encapsulating it in a function.

Reply:
I will keep this as a module for now. Thanks!